cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -153,6 +156,9 @@
   private static LocationFactory locationFactory;
   private static File adapterDir;
 
+  protected static MetricStore metricStore;
```

Comment:
why protected in AppFabricTestBase? Can you make them private in ``WorkflowSLAHttpHandlerTest``, since that's the only place they are used?